### PR TITLE
[18.0][MIG] delivery_package_fee: Migration to 18.0

### PR DESCRIPTION
**Pull request by chaule97**
- From Odoo 17.0+, stock was separated from delivery, in https://github.com/odoo/odoo/commit/b2541e6218e014a3ed1f2b58915ad897eb2a3f58, hence delivery_package_fee depends on stock_delivery
- From Odoo 18.0, `is_storable` was introduced for simpler tracking instead of type `product`, in https://github.com/odoo/odoo/commit/83d0f2bb33765f8840821c11a269c48ca28a573.
- From Odoo 17.0, Public pricelist was removed in https://github.com/odoo/odoo/commit/40f0482b680a7872d338fa8fd10eec4663637b46, adding pricelist for testing.
- https://github.com/odoo/odoo/pull/137864, we don't have qty_done anymore, we will use quantity and picked.

### CODE REVIEW

File: delivery_package_fee/readme/DESCRIPTION.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+Add delivery fees on Sales Orders based on the delivered packages.
+
+A list of Package Fees can be added on shipping methods. When a outgoing
+transfer is done, for each package fee configured on the shipping
```
Comment (jbaudoux) [inline delivery_package_fee/readme/DESCRIPTION.md]:
```suggestion
A list of Package Fees can be configured on delivery methods. When a outgoing
transfer is done, for each package fee configured on the delivery
```

File: delivery_package_fee/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,14 @@
+# Copyright 2020 Camptocamp
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+{
+    "name": "Delivery Package Fees",
+    "summary": "Add fees on delivered packages on shipping methods",
```
Comment (jbaudoux) [inline delivery_package_fee/__manifest__.py]:
```suggestion
    "summary": "Add fees on sales order for delivered packages",
```

File: delivery_package_fee/readme/DESCRIPTION.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+method, a new sale order line is created with:
```
Comment (jbaudoux) [inline delivery_package_fee/readme/DESCRIPTION.md]:
```suggestion
method, a new sales order line is created with:
```

**Review by jbaudoux — COMMENTED** [review]
LG
Some rewording

**Review by jbaudoux — APPROVED** [review]

**Review by ajaniszewska-dev — APPROVED** [review]

**Pull request merged**